### ISSUE TITLE
Why don't you calculate the speaker consistency loss in the first 1000 steps?

### ISSUE THREAD

**Issue by isletennos** [ConsistencyVC/ConsistencyVC-voive-conversion#11]
As stated in the subject, 
why don't you calculate the speaker consistency loss in the first 1000 steps?

Thank you.

**Comment by ConsistencyVC** [comment]
In Section 2 of [YourTTS paper](https://arxiv.org/abs/2112.02418v1), they have defined the Speaker Consistency Loss (SCL) function. In addition, they have used this loss function on 4 **fine-tuning** experiments in Sections 3 and 4 (EXP. 1 + SCL, EXP. 2 + SCL, EXP. 3 + SCL, and EXP. 4 + SCL).
So I refer to YourTTS, use the consistency loss to fine-tune the model.

**Comment by ConsistencyVC** [comment]
ディープラーニングは時に形而上学的で、その理由を数学的に説明する方法がない（悲）
